pkg/signup/service/signup_service.go

```diff
@@ -120,6 +130,11 @@ func (s *ServiceImpl) newUserSignup(ctx *gin.Context) (*toolchainv1alpha1.UserSi
 			OriginalSub:   ctx.GetString(context.OriginalSubKey),
 		},
 	}
+
+	if captchaScore > -1.0 {
```

Comment:
Why `-1.0`? Is `-1.0` some magic captcha? What does captcha =< -1.0 mean?

Reply:
Now I see how it works below. But let's add the comment to the `IsPhoneVerificationRequired()` stating that the function either returns the actual captcha score or -1 with explanation what -1 means.

Reply:
Good point, added a comment in c37ee1c084857f2f92731fe33948ab6dbc514b2e